crates/bevy_ui/src/render/mod.rs

```diff
@@ -305,15 +306,16 @@ pub fn extract_text_uinodes(
                 let rect = atlas.textures[index];
                 let atlas_size = Some(atlas.size);
 
-                let transform =
+                let transform = Transform::from(*global_transform);
```

Comment:
In terms of code readability, I feel like converting the GlobalTransform to a Transform is non-obvious. It would make more sense to me to extract the scale, rotation, translation from the GlobalTransform itself. I understand that this is using Affine3A::to_scale_rotation_translation() under the hood but the code one sees here in a situation like this doesn't really make sense.

Reply:
Swapped this out for `GlobalTransform::to_scale_rotation_translation`